### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ayush-Thakur-geek**
translated a non-translated word.

Improvement to translation.

### CODE REVIEW

File: localization/hi/abstract-document/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ tag:
 प्रकार-सुरक्षा का समर्थन। पैटर्न एक वर्ग के विभिन्न गुणों को अलग-अलग करने के लिए लक्षणों का उपयोग करता है
 इंटरफेस।
 
-**Programmatic Example**
+**प्रोग्रामेटिक उदाहरण**
```
Comment (yoavami) [inline localization/hi/abstract-document/README.md:34]: Changing this seems appropriate

File: localization/hi/abstract-document/README.md [hunk 2]
```diff
@@ -184,4 +184,4 @@ public class Car extends AbstractDocument implements HasModel, HasPrice, HasPart
 
 * [Wikipedia: Abstract Document Pattern](https://en.wikipedia.org/wiki/Abstract_Document_Pattern)
 * [Martin Fowler: Dealing with properties](http://martinfowler.com/apsupp/properties.pdf)
-* [Pattern-Oriented Software Architecture Volume 4: A Pattern Language for Distributed Computing (v. 4)](https://www.amazon.com/gp/product/0470059028/ref=as_li_qf_asin_il_tl?ie=UTF8&tag=javadesignpat-20&creative=9325&linkCode=as2&creativeASIN=0470059028&linkId=e3aacaea7017258acf184f9f3283b492)
\ No newline at end of file
+* [Pattern-Oriented Software Architecture Volume 4: A Pattern Language for Distributed Computing (v. 4)](https://www.amazon.com/gp/product/0470059028/ref=as_li_qf_asin_il_tl?ie=UTF8&tag=javadesignpat-20&creative=9325&linkCode=as2&creativeASIN=0470059028&linkId=e3aacaea7017258acf184f9f3283b492)
```
Comment (yoavami) [inline localization/hi/abstract-document/README.md:187]: This is a great resource.

**Review by yoavami — APPROVED** [review]
Changes look good.

**Comment by Ayush-Thakur-geek** [comment]
review and comment when possible!

File: localization/hi/abstract-document/README.md (hunk 1 above)
Comment (Ayush-Thakur-geek) [reply localization/hi/abstract-document/README.md:34]: So could you pls merge it to the main branch.

**Comment by Ayush-Thakur-geek** [comment]
If the changes are right, can someone merge my pr to the main branch

**Review by iluwatar — APPROVED** [review]

**Comment by iluwatar** [comment]
@all-contributors please add @Ayush-Thakur-geek for translation

**Pull request merged**